index.html

```diff
@@ -0,0 +1,267 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./style.css" />
+  <title>PERIOD SPONSORS!</title>
+
+</head>
+
+<body>
+  <!-- video or image as a header is cool :) -->
+  <header>
+    <section class="hero-wrapper">
+      <div class="hero-image">
+        <div class="hero-text">
+          <h1 class="hero-h1">MENSTRU-AID</h1>
+          <p class="hero-p">Fight <span class="red">period poverty</span>, become a <span class="red">period
+              sponsor!</span>
+          </p>
+          <a href="#the-form"><button>Become a <span class="red">period sponsor</span></button></a>
+        </div>
+      </div>
+      </div>
+    </section>
+
+
+    <nav>
+      <a href="#manifesto">Manifest</a>
+      <a href="#accordion">THE PROJECT</a>
+    </nav>
+  </header>
+
+  <!-- MANIFESTO -->
+  <section class="manifesto" id="manifesto">
+    <h2 class="manifesto-h2">
+      "Every day we meet young <span class="red">menstruators</span> who cannot
+      afford <span class="red">menstrual hygine</span> products"
+    </h2>
+    <h3 class="manifesto-h3">
+      / Göteborg's Stadsmission
+    </h3>
+
+    <div class="p-manifesto-wrapper">
+      <p class="p-manifesto">Once a month. Twelve months a year. For many, it is everyday-life to buy a package of pads,
+        disinfect a menstrual cup or throw a tampon in the bag. But not for everyone </p>
+    </div>
+  </section>
```

Comment:
It's really effectual working with spans and different font-sizes but have in mind that it affects the readability